python/simulators/simulators.cpp

```diff
@@ -1,125 +0,0 @@
-/*
```

Comment:
I thinks this file and the corresponding `simulators.hpp` file are not used anymore. Probably I forgot to delete them in a previous PR.